Libraries/CustomComponents/Navigator/NavigatorNavigationBar.js

```diff
@@ -145,6 +145,7 @@ var NavigatorNavigationBar = React.createClass({
       var component = this._components[componentName].get(this.props.navState.routeStack[index]);
       var props = this._getReusableProps(componentName, index);
       if (component && interpolate[componentName](props.style, amount)) {
+        props.pointerEvents = props.style.opacity === 0 ? 'none' : 'box-none';
```

Comment:
I'm not really happy with this line because the check happens on every iteration in a tight loop. But I'm not sure where a better place to put it would be; the NavigationBar's `props` don't appear to be updated ever with the final route after a transition has taken place, so I can't make it contingent on that.